lib/index.js

```diff
@@ -20,9 +20,11 @@ var AdRoll = module.exports = integration('AdRoll')
   .global('__adroll_loaded')
   .global('adroll_adv_id')
   .global('adroll_custom_data')
+  .global('adroll_email')
   .global('adroll_pix_id')
   .option('advId', '')
   .option('pixId', '')
+  .option('v1', false)
```

Comment:
Okay yeah, I'm calling it—let's start prefixing internal settings with a leading underscore.